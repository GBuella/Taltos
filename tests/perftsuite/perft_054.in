setboard 7k/RR6/8/8/8/8/rr6/7K w - - 0 1
echo 7k/RR6/8/8/8/8/rr6/7K w - - 0 1
perft 1
perft 2
perft 3
perft 4
perft 5
perft 6
