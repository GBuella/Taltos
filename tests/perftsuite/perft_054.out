7k/RR6/8/8/8/8/rr6/7K w - - 0 1
19
275
5300
104342
2161211
44956585
